stan/math/prim/mat/fun/accumulator.hpp

```diff
@@ -63,7 +63,7 @@ class accumulator {
    * @tparam S Type of argument
    * @param x Value to add
    */
-  template <typename S, typename = enable_if_not_arithmetic<S>,
+  template <typename S, typename = disable_if_arithmetic<S>,
```

Comment:
The standard veto'd `disable_if`s being in the standard but boost has them and imo I prefer `disable_if` to `enable_if_not`